WARNING: You are using pip version 21.1.2; however, version 21.3.1 is available.
You should consider upgrading via the '/apps/t3/sles12sp4/free/jupyterlab/2.1.0/gcc4.8.5/bin/python3 -m pip install --upgrade pip' command.
Traceback (most recent call last):
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/mpi_ops.py", line 33, in <module>
    from horovod.torch import mpi_lib_v2 as mpi_lib
ImportError: /home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/mpi_lib_v2.cpython-36m-x86_64-linux-gnu.so: undefined symbol: _ZNK2at6Tensor4div_ERKN3c106ScalarE

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "zzz3.py", line 12, in <module>
    import horovod.torch as hvd
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/__init__.py", line 35, in <module>
    from horovod.torch import elastic
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/elastic/__init__.py", line 17, in <module>
    from horovod.torch.mpi_ops import init, shutdown
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/mpi_ops.py", line 35, in <module>
    check_installed_version('pytorch', torch.__version__, e)
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/common/util.py", line 260, in check_installed_version
    raise HorovodVersionMismatchError(name, version, installed_version) from exception
horovod.common.exceptions.HorovodVersionMismatchError: Framework pytorch installed with version 1.9.0+cu102 but found version 1.10.1+cu102.
             This can result in unexpected behavior including runtime errors.
             Reinstall Horovod using `pip install --no-cache-dir` to build with the new version.
Traceback (most recent call last):
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/mpi_ops.py", line 33, in <module>
    from horovod.torch import mpi_lib_v2 as mpi_lib
ImportError: /home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/mpi_lib_v2.cpython-36m-x86_64-linux-gnu.so: undefined symbol: _ZNK2at6Tensor4div_ERKN3c106ScalarE

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "zzz3.py", line 12, in <module>
    import horovod.torch as hvd
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/__init__.py", line 35, in <module>
    from horovod.torch import elastic
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/elastic/__init__.py", line 17, in <module>
    from horovod.torch.mpi_ops import init, shutdown
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/mpi_ops.py", line 35, in <module>
    check_installed_version('pytorch', torch.__version__, e)
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/common/util.py", line 260, in check_installed_version
    raise HorovodVersionMismatchError(name, version, installed_version) from exception
horovod.common.exceptions.HorovodVersionMismatchError: Framework pytorch installed with version 1.9.0+cu102 but found version 1.10.1+cu102.
             This can result in unexpected behavior including runtime errors.
             Reinstall Horovod using `pip install --no-cache-dir` to build with the new version.
Traceback (most recent call last):
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/mpi_ops.py", line 33, in <module>
    from horovod.torch import mpi_lib_v2 as mpi_lib
ImportError: /home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/mpi_lib_v2.cpython-36m-x86_64-linux-gnu.so: undefined symbol: _ZNK2at6Tensor4div_ERKN3c106ScalarE

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "zzz3.py", line 12, in <module>
    import horovod.torch as hvd
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/__init__.py", line 35, in <module>
    from horovod.torch import elastic
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/elastic/__init__.py", line 17, in <module>
    from horovod.torch.mpi_ops import init, shutdown
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/mpi_ops.py", line 35, in <module>
    check_installed_version('pytorch', torch.__version__, e)
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/common/util.py", line 260, in check_installed_version
    raise HorovodVersionMismatchError(name, version, installed_version) from exception
horovod.common.exceptions.HorovodVersionMismatchError: Framework pytorch installed with version 1.9.0+cu102 but found version 1.10.1+cu102.
             This can result in unexpected behavior including runtime errors.
             Reinstall Horovod using `pip install --no-cache-dir` to build with the new version.
Traceback (most recent call last):
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/mpi_ops.py", line 33, in <module>
    from horovod.torch import mpi_lib_v2 as mpi_lib
ImportError: /home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/mpi_lib_v2.cpython-36m-x86_64-linux-gnu.so: undefined symbol: _ZNK2at6Tensor4div_ERKN3c106ScalarE

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "zzz3.py", line 12, in <module>
    import horovod.torch as hvd
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/__init__.py", line 35, in <module>
    from horovod.torch import elastic
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/elastic/__init__.py", line 17, in <module>
    from horovod.torch.mpi_ops import init, shutdown
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/mpi_ops.py", line 35, in <module>
    check_installed_version('pytorch', torch.__version__, e)
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/common/util.py", line 260, in check_installed_version
    raise HorovodVersionMismatchError(name, version, installed_version) from exception
horovod.common.exceptions.HorovodVersionMismatchError: Framework pytorch installed with version 1.9.0+cu102 but found version 1.10.1+cu102.
             This can result in unexpected behavior including runtime errors.
             Reinstall Horovod using `pip install --no-cache-dir` to build with the new version.
Traceback (most recent call last):
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/mpi_ops.py", line 33, in <module>
    from horovod.torch import mpi_lib_v2 as mpi_lib
ImportError: /home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/mpi_lib_v2.cpython-36m-x86_64-linux-gnu.so: undefined symbol: _ZNK2at6Tensor4div_ERKN3c106ScalarE

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "zzz3.py", line 12, in <module>
    import horovod.torch as hvd
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/__init__.py", line 35, in <module>
    from horovod.torch import elastic
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/elastic/__init__.py", line 17, in <module>
    from horovod.torch.mpi_ops import init, shutdown
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/mpi_ops.py", line 35, in <module>
    check_installed_version('pytorch', torch.__version__, e)
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/common/util.py", line 260, in check_installed_version
    raise HorovodVersionMismatchError(name, version, installed_version) from exception
horovod.common.exceptions.HorovodVersionMismatchError: Framework pytorch installed with version 1.9.0+cu102 but found version 1.10.1+cu102.
             This can result in unexpected behavior including runtime errors.
             Reinstall Horovod using `pip install --no-cache-dir` to build with the new version.
Traceback (most recent call last):
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/mpi_ops.py", line 33, in <module>
    from horovod.torch import mpi_lib_v2 as mpi_lib
ImportError: /home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/mpi_lib_v2.cpython-36m-x86_64-linux-gnu.so: undefined symbol: _ZNK2at6Tensor4div_ERKN3c106ScalarE

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "zzz3.py", line 12, in <module>
    import horovod.torch as hvd
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/__init__.py", line 35, in <module>
    from horovod.torch import elastic
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/elastic/__init__.py", line 17, in <module>
    from horovod.torch.mpi_ops import init, shutdown
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/mpi_ops.py", line 35, in <module>
    check_installed_version('pytorch', torch.__version__, e)
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/common/util.py", line 260, in check_installed_version
    raise HorovodVersionMismatchError(name, version, installed_version) from exception
horovod.common.exceptions.HorovodVersionMismatchError: Framework pytorch installed with version 1.9.0+cu102 but found version 1.10.1+cu102.
             This can result in unexpected behavior including runtime errors.
             Reinstall Horovod using `pip install --no-cache-dir` to build with the new version.
Traceback (most recent call last):
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/mpi_ops.py", line 33, in <module>
    from horovod.torch import mpi_lib_v2 as mpi_lib
ImportError: /home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/mpi_lib_v2.cpython-36m-x86_64-linux-gnu.so: undefined symbol: _ZNK2at6Tensor4div_ERKN3c106ScalarE

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "zzz3.py", line 12, in <module>
    import horovod.torch as hvd
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/__init__.py", line 35, in <module>
    from horovod.torch import elastic
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/elastic/__init__.py", line 17, in <module>
    from horovod.torch.mpi_ops import init, shutdown
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/mpi_ops.py", line 35, in <module>
    check_installed_version('pytorch', torch.__version__, e)
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/common/util.py", line 260, in check_installed_version
    raise HorovodVersionMismatchError(name, version, installed_version) from exception
horovod.common.exceptions.HorovodVersionMismatchError: Framework pytorch installed with version 1.9.0+cu102 but found version 1.10.1+cu102.
             This can result in unexpected behavior including runtime errors.
             Reinstall Horovod using `pip install --no-cache-dir` to build with the new version.
Traceback (most recent call last):
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/mpi_ops.py", line 33, in <module>
    from horovod.torch import mpi_lib_v2 as mpi_lib
ImportError: /home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/mpi_lib_v2.cpython-36m-x86_64-linux-gnu.so: undefined symbol: _ZNK2at6Tensor4div_ERKN3c106ScalarE

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "zzz3.py", line 12, in <module>
    import horovod.torch as hvd
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/__init__.py", line 35, in <module>
    from horovod.torch import elastic
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/elastic/__init__.py", line 17, in <module>
    from horovod.torch.mpi_ops import init, shutdown
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/torch/mpi_ops.py", line 35, in <module>
    check_installed_version('pytorch', torch.__version__, e)
  File "/home/1/16B14626/.local/lib/python3.6/site-packages/horovod/common/util.py", line 260, in check_installed_version
    raise HorovodVersionMismatchError(name, version, installed_version) from exception
d
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
mpirun detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[4528,1],1]
  Exit code:    1
--------------------------------------------------------------------------
